TROUBLESHOOTING.md

```diff
@@ -0,0 +1,23 @@
+# Troubleshooting
+This document covers some common issues and how to resolve them. You should always follow these steps before opening a new issue.
+
+## Merge Blocking
+By making a change to the test configuration that adds, removes or renames a merge-blocking status will result in a merge queue that is stuck unless manual steps with the [migratestatus](https://github.com/kubernetes/test-infra/tree/master/maintenance/migratestatus) tool.
+
+### Add
+When a new prow job is added, you can include the new status check in the PR by commenting `/retest`.
```

Comment:
We want to use the `commenter` robot to `/test foo` on all open PRs automatically